File: 20 - Multistate Sales Tax Calculator/Python/main.py
#!/usr/bin/env python

def main():
    amount = float(input("What is the order amount? "))
    tax    = 0
    state  = input("What state do you live in? ").lower()

    if state == "wisconsin":
        county = input("What county do you live in? ").lower()
        if county == "eau claire":
            tax = 0.005
        elif county == "dunn":
            tax = 0.004
    elif state == "illinois":
        tax = 0.08

    print("The tax is ${}.\nThe total is ${}.\n".format(amount * tax, amount +
                                                      (amount * tax)))

if __name__ == "__main__":
    main()

File: 20 - Multistate Sales Tax Calculator/Golang/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strings"
)

func main() {
	scanner := bufio.NewScanner(os.Stdin)

	var amount, tax float64
	fmt.Print("What is the order amount? ")
	fmt.Scanf("%f", &amount)

	fmt.Print("What state do you live in? ")
	scanner.Scan()
	state := strings.ToLower(scanner.Text())

	if state == "wisconsin" {
		fmt.Print("What county do you live in? ")
		scanner.Scan()
		county := strings.ToLower(scanner.Text())
		if county == "eau claire" {
			tax = 0.005
		} else if county == "dunn" {
			tax = 0.004
		}
	} else if state == "illinois" {
		tax = 0.008
	}

	fmt.Printf("The tax is $%.2f.\nThe total is $%.2f.\n", amount*tax, amount+(amount*tax))
}
